x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/find_maintenance_windows.ts

```diff
@@ -90,6 +90,70 @@ export default function findMaintenanceWindowTests({ getService }: FtrProviderCo
               throw new Error(`Scenario untested: ${JSON.stringify(scenario)}`);
           }
         });
+
+        it('should handle find maintenance window request with pagination', async () => {
+          const { body: createdMaintenanceWindow1 } = await supertest
+            .post(`${getUrlPrefix(space.id)}/internal/alerting/rules/maintenance_window`)
+            .set('kbn-xsrf', 'foo')
+            .send(createParams);
+
+          const { body: createdMaintenanceWindow2 } = await supertest
+            .post(`${getUrlPrefix(space.id)}/internal/alerting/rules/maintenance_window`)
+            .set('kbn-xsrf', 'foo')
+            .send(createParams);
```

Comment:
nit: Could you please change the `title` of the MW so we are sure that in L147 the correct one is being returned.